pkg/netpol/connlist/connlist.go

```diff
@@ -400,6 +407,38 @@ func (ca *ConnlistAnalyzer) getConnectionsList(pe *eval.PolicyEngine) ([]Peer2Pe
 			res = append(res, connectionObj)
 		}
 	}
+
+	// analyze ingress connections - create connection objects for relevant ingress analyzer connections
+	if !ia.IsEmpty() {
+		ingressConns := ia.AllowedIngressConnections()
+		// adding the ingress controller pod to the policy engine,
+		ingressControllerPod, err := pe.AddPodByNameAndNamespace(ingressanalyzer.IngressPodName, ingressanalyzer.IngressPodNs)
+		if err != nil {
+			ca.errors = append(ca.errors, newResourceEvaluationError(err))
+			return nil, err
+		}
+		for _, peer := range peerList {
+			// if there is no ingress connections to the peer - skip
+			if _, ok := ingressConns[peer.String()]; !ok {
+				continue
+			}
+			// otherwise, compute allowed connections based on pe.policies to the peer, then intersect the conns with
+			// ingress connections to the peer -> the intersection will be appended to the result
+			peConn, err := pe.AllAllowedConnectionsBetweenWorkloadPeers(ingressControllerPod, peer)
+			if err != nil {
+				ca.errors = append(ca.errors, newResourceEvaluationError(err))
+				return nil, err
+			}
+			ingressConns[peer.String()].ConnectionsIntersection(peConn)
+			connectionObj := &connection{
+				src:               ingressControllerPod,
+				dst:               peer,
+				allConnections:    ingressConns[peer.String()].AllConnections(),
+				protocolsAndPorts: ingressConns[peer.String()].ProtocolsAndPortsMap(),
+			}
+			res = append(res, connectionObj)
```

Comment:
```suggestion
			res = append(res, p2pConnection)
```

Reply:
do we want to keep in the output a line for `no connection` case? 
or instead issue a warning about a connection specified by ingress/route resource that is blocked by network policies?